src/System.Private.Xml/src/System/Xml/Core/HtmlUtf8RawTextWriter.cs

```diff
@@ -711,7 +711,7 @@ private unsafe void WriteUriAttributeText(char* pSrc, char* pSrcEnd)
                             break;
                         default:
                             const string hexDigits = "0123456789ABCDEF";
-                            fixed (byte* pUriEscapingBuffer = _uriEscapingBuffer)
+                            fixed (byte* pUriEscapingBuffer = &_uriEscapingBuffer[0])
```

Comment:
We know for certain that _uriEscapingBuffer has been initialized at this point?  (The change arguably makes things better if it hasn't, in that it essentially introduces validation to avoid producing a null pUriEscapingBuffer.)